project/LinkerdBuild.scala

```diff
@@ -522,6 +542,29 @@ object LinkerdBuild extends Base {
       .settings(inConfig(OpenJ9)(OpenJ9Settings))
       .configDependsOn(Dcos)(dcosBootstrap)
       .settings(inConfig(Dcos)(DcosSettings))
+      .configDependsOn(NoZk)(BundleProjectsNoZk: _*)
+      .settings(inConfig(NoZk)(NoZkSettings))
+      .settings(
+        assembly := (assembly in Bundle).value,
+        docker := (docker in Bundle).value,
+        dockerBuildAndPush := (dockerBuildAndPush in Bundle).value,
+        dockerPush := (dockerPush in Bundle).value
+      )
+
+    val noZk = aggregateDir("namerd",
```

Comment:
should this project just use the NoZk config only?

Reply:
Updated the PR with this change